crates/api/api/src/local_user/add_admin.rs

```diff
@@ -24,6 +24,11 @@ pub async fn add_admin(
   // Make sure user is an admin
   is_admin(&local_user_view)?;
 
+  // Cant use this endpoint on yourself (use `/api/v4/admin/leave` instead)
+  if local_user_view.person.id == data.person_id {
+    Err(LemmyErrorType::CannotLeaveAdmin)?
+  }
```

Comment:
But you deleted the `leave_admin` endpoint. 

- So you should this check under `if !data.added` 
- Add the *at least one admin* check here also.
- Also change this comment